controller/build/build.go

```diff
@@ -23,43 +23,137 @@ import (
 	dockeropts "github.com/docker/cli/opts"
 	"github.com/docker/go-units"
 	"github.com/moby/buildkit/client"
+	"github.com/moby/buildkit/client/llb"
 	"github.com/moby/buildkit/session/auth/authprovider"
 	"github.com/moby/buildkit/util/grpcerrors"
 	"github.com/pkg/errors"
+	"golang.org/x/sync/errgroup"
 	"google.golang.org/grpc/codes"
 )
 
-const defaultTargetName = "default"
+const DefaultTargetName = "default"
```

Comment:
We don't need to make it public.

Reply:
Yes indeed, this is an oversight related to previous changes.